### ISSUE TITLE
Hypoplastic mesencephalon

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7527]
**Preferred term label:**
Hypoplastic mesencephalon

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
616202 
PMID: 25561519  2/6

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
done
